libs/builder/interfaces/asset.ts

```diff
@@ -4,5 +4,5 @@ export interface NgDocAsset {
 	filePath: string;
 	isEmpty: boolean;
 	icon?: string;
-	lang: string
```

Comment:
Could you please remove from the PR all the files that ended up here due to code formatting? I will fix the formatting, and most likely switch to `indent_size = 2` instead of `indent_size = 4`.